example/spec/loadTest/collection/s3_loadtest_MOD09GQ_006L.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "MOD09GQ",
```

Comment:
Does the arrow indicate that you used tabs?

Reply:
Hmm, the editor isn't setup to do that, however this was directly copied from another file.... so probably.   Will sort.

Reply:
e6cb0c98c

Reply:
Oops, also 87f14a05c